packages/chat-component-bindings/package.json

```diff
@@ -40,7 +40,7 @@
     "memoize-one": "^5.2.1"
   },
   "peerDependencies": {
-    "@azure/communication-chat": "1.4.0-beta.1 || >=1.3.2",
```

Comment:
It looks like communication-react package.json has incorrect version too

Reply:
I just took a look on main and it looks to be pointing to `1.4.0-beta.2` Can you verify?